src/System.Net.Security/src/System/Net/Security/SecureChannel.cs

```diff
@@ -642,7 +648,6 @@ private bool AcquireServerCredentials(ref byte[] thumbPrint, byte[] clientHello)
                     throw new AuthenticationException(SR.net_ssl_io_no_server_cert);
                 }
             }
-            // This probably never gets called as this is a client options delegate
             else if (_sslAuthenticationOptions.CertSelectionDelegate != null)
             {
                 X509CertificateCollection tempCollection = new X509CertificateCollection();
```

Comment:
Add explicit comment why we pass string.Empty (compat with .NET Framework) instead of hostName

Reply:
Added.